### ISSUE TITLE
### Icon Request - Spyder 

### ISSUE THREAD

**Issue by OnTheRicky** [snwh/paper-icon-theme#370]
### Icon Request

- Spyder
- Spyder is an open source cross-platform IDE for scientific programming in the Python language
- ![spyder](https://cloud.githubusercontent.com/assets/19494307/18461926/90862250-797e-11e6-9916-426b492d13d7.png)

**Comment by snwh** [comment]
I'll need the icon filename for this app.

**Comment by OnTheRicky** [comment]
I'm not sure where to find it, since it's not in /usr/share/applications

**Comment by snwh** [comment]
It doesn't have a `.desktop` file?

**Comment by OnTheRicky** [comment]
Hi Sam,

The .desktop files I have here are spyder.desktop and spyder3.desktop,
which I assumed relate to the two different versions of spyder I have
installed for python 2.7 and 3.5 respectively.

**Comment by snwh** [comment]
Right, the desktop file should refer to the icon with an `Icon=` field.

**Comment by OnTheRicky** [comment]
Icon=spyder 
(for python 2.7)

and 
Icon=spyder3
(for python 3.5)

Thanks for the help!

**Comment by tabakisp** [comment]
`Spyder` [.desktop file](https://apt-browse.org/browse/ubuntu/trusty/universe/all/spyder/2.2.5+dfsg-1/file/usr/share/applications/spyder.desktop)
`Spyder3` [.desktop file](https://apt-browse.org/browse/debian/jessie/main/all/spyder3/2.3.1+dfsg-1/file/usr/share/applications/spyder3.desktop)
